src/map/pet.cpp

```diff
@@ -1213,14 +1220,26 @@ int32 pet_select_egg(map_session_data *sd,short egg_index)
  * @param sd : player requesting
  * @param item_id : item ID of the taming item used
  */
-void pet_catch_process1(map_session_data& sd, t_itemid item_id)
-{
+void pet_catch_process_start( map_session_data& sd, t_itemid item_id, e_pet_catch_flag flag ){
 	if (map_getmapflag(sd.bl.m, MF_NOPETCAPTURE)) {
 		clif_displaymessage(sd.fd, msg_txt(&sd, 669)); // You can't catch any pet on this map.
 		return;
 	}
 
-	sd.taming_item = item_id;
+	std::shared_ptr<s_pet_catch_process> process = util::umap_find( pet_catchprocesses, sd.status.char_id );
+
+	if( process != nullptr ){
+		// TODO: Just delete?
```

Comment:
I guess add a return for now?

Reply:
The question is: How does AEGIS react, if you use another taming item, while you have the other pet catching process open?

Reply:
Oh I already tested that, it restarts the process with the new item